doc/running.rst

```diff
@@ -46,6 +46,40 @@ environment variable, such as in the following example::
    $ srun -n 512 python -m mpi4py examples/wave-eager-mpi.py'
 
 
+OpenCL device selection
+-----------------------
+
+The :mod:`pyopencl` package supports selecting the device on which to run OpenCL code.
+There are multiple ways in which this selection can happen:
+
+- **Interactively.** When running mirgecom on the command line (i.e., not in a
+  batch script), it will present a (potentially multiple-choice) dialog with
+  the available OpenCL drivers (such as pocl) and the device (such as CPU or
+  GPU). After a device has been selected, pyopencl will print the chosen
+  device in the form of a value for the ``PYOPENCL_CTX`` environment
```

Comment:
Use `PYOPENCL_TEST` over `PYOPENCL_CTX`, which also works with the pytest functions (in addition to `create_some_context`)

Reply:
Changed